appveyor.yml

```diff
@@ -110,10 +111,11 @@ build_script:
  - ps: |
      cmd.exe /c "scons checkPot $sconsArgs"
      if($LastExitCode -ne 0) {
-      $errorCode=$LastExitCode
-      Add-AppveyorMessage "Translation comments missing or unexpectedly included."
+      Set-AppveyorBuildVariable "testFailExitCode" $LastExitCode
+      Add-AppveyorMessage "FAIL: Translation comments check. Translation comments missing or unexpectedly included. See build log for more information."
+     } else {
+      Add-AppveyorMessage "PASS: Translation comments check."
      }
-     if ($errorCode -ne 0) { $host.SetShouldExit($errorCode) }

```

Comment:
I think it might even help reduce time overall as if a build has an issue that causes a later failure as well it will be caught early. A checkpot failure should appear in the same time frame if you are watching the build messages.